governance-exercises/CivicSage Exercises/Ex01/1. readme.md

```diff
@@ -0,0 +1,38 @@
+## Determine what course of action you would take as a Governance Facilitator and communicate it clearly
+
+1. Review the poll request and related discussion to determine if everything is clear.
+
+2. Clarify with Alice and Faraz the addition of Debt Increase Cooldown parameter, as it was not included in Alice’s post, but only mentioned in the reply from Faraz. I assume Faraz refers to the Ceiling Increase Cooldown. Post clarification in the forum thread.
+
+3. Separate the request into three polls:
+    - Adjust Spork D3M Debt parameters
+        - Decision impacting directly Maker Core
+        - A recurring operational decision to be managed via Weekly Governance Cycle
+        - Poll template to be used: *Adjust $protocol_name D3M Parameters - $date_MONTH_DD,_YYYY.md*
+    - Increase Spork DAI deposit rate
+        - Decision unrelated to D3M Debt parameters or ABC token
+        - A recurring operational decision to be managed via Weekly Governance Cycle
+        - Poll template to be used: *$GenericBinary - $date_MONTH_DD,_YYYY.md*
+    - Onboarding of ABC token to Spork Protocol
+        - Decision seems to be internal to subDAO governance (new process to be followed after subDAO token launch?)
+        - Following MIP16c3, being a decision with long-term impact and no urgency, this request should be managed via Monthly Governance Cycle. Nevertheless, the relevant poll template (*Onboard $token to Spark Protocol - $date_MONTH_DD,_YYYY.md*) indicates Weekly Governance Cycle.
+
+4. Make a forum post explaining the separation of request into three polls and the rationale behind it. Had we not had the reply from Bob and Faraz, I would have split the request from Alice into three forum posts. In this case, I’d leave the post as it is and only split the polls.
+
+5. Make a forum post, explicitly approving the three poll requests, based on:
+    - No/ low risk for Maker Ecosystem/ community.
```

Comment:
Keep in mind that most of the time we won't be able to evaluate the risk as we're not subject matter experts. In this post, you will probably have someone from the stability scope advisor (BA Labs) do the risk assessment and you can use their suggestions